packages/shared/src/components/search/SearchResults/SearchResultsLayout.tsx

```diff
@@ -91,9 +97,18 @@ export const SearchResultsLayout = (
     <section className="mx-auto w-full laptopL:max-w-screen-laptop">
       <div className="flex flex-row border-border-subtlest-tertiary laptop:-mx-8 laptop:pb-0 laptopL:mx-auto laptopL:border-x">
         <div className="flex-1 border-r border-border-subtlest-tertiary">
-          <h2 className="px-4 py-4 font-bold text-text-primary typo-body">
-            Related posts
-          </h2>
+          <div className="flex items-center justify-between">
+            <h2 className="px-4 py-4 font-bold text-text-primary typo-body">
+              Related posts
+            </h2>
+            {searchVersion >= 3 ? (
+              <div className="mx-4 flex gap-2">
+                <SearchFilterTimeButton />
+                <SearchFilterPostTypeButton />
+              </div>
+            ) : undefined}
```

Comment:
```suggestion
            {searchVersion >= 3 && (
              <div className="mx-4 flex gap-2">
                <SearchFilterTimeButton />
                <SearchFilterPostTypeButton />
              </div>
            )}
```

Reply:
Think I always prefer the more explicit over the && because it can also evaluate null and just render null.

Let me know what you thnk